prime-router/metadata/hl7_mapping/common/datatype/ce-coded-with-exceptions.yml

```diff
@@ -17,4 +20,14 @@ elements:
   - name: coding-system-alternate-version-id
     resource: '%resource.extension(%`rsext-alternate-identifier`).value.coding.extension(%`rsext-alternate-version`)'
     value: [ '%resource.value' ]
-    hl7Spec: [ '%{cweFieldPath}-8' ]
\ No newline at end of file
+    hl7Spec: [ '%{cweFieldPath}-8' ]
+
+  - name: original-text
+    condition: '%alternateIdentifierExtension.exists() and %originalTextExtension.exists()'
+    value: [ '%originalTextExtension.value' ]
+    hl7Spec: [ '%{cweFieldPath}-9' ]
+
+  - name: original-text-from-display
+    condition: '%alternateIdentifierExtension.exists().not() and %originalTextExtension.exists().not()'
+    value: [ '%resource.coding.display' ]
+    hl7Spec: [ '%{cweFieldPath}-9' ]
```

Comment:
CA wants original text populated. Might be good to include this for every receiver.